cookieplone/cli.py

```diff
@@ -38,6 +38,10 @@ def parse_extra_content(value: list[str]) -> dict:
     return dict([s.split("=") for s in value])
 
 
+def parse_boolean(value: str) -> bool:
```

Comment:
I'd rename and refactor throughout to more accurately reflect what it does.

```suggestion
def parse_yes_values(value: str) -> bool:
```

Reply:
@stevepiercy No thanks. It also parses strings that contain values which represent "no".